tests/testthat/test-admiral_options.R

```diff
@@ -0,0 +1,29 @@
+# get_admiral_option ----
+## Test 1: get works ----
+test_that("get_admiral_option Test 1: get works", {
+  expect_equal(get_admiral_option("subject_keys"), vars(STUDYID, USUBJID))
+})
+
+## Test 2: common typo gives error ----
+test_that("get_admiral_option Test 2: common typo gives error", {
+  expect_error(get_admiral_option("subject_key"))
+})
+
+## Test 3: unexpected function input for get gives error ----
+test_that("get_admiral_option Test 3: unexpected function input for get gives error", {
+  expect_error(get_admiral_option(subject_keys))
```

Comment:
This test does not test the function because `subject_keys` does not exist:
```
Error in assert_character_scalar(option) : 
  object 'subject_keys' not found
```